schema/course.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$anchor": "youtube-course",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$patch": {
+    "source": {
+      "$ref": "course"
+    },
+    "with": {
+      "properties": {
+        "_globals": {
+          "type": "object",
+          "default": {},
+          "properties": {
+            "_youtube": {
+              "type": "object",
+              "default": {},
+              "properties": {
+                "ariaRegion": {
+                  "type": "string",
+                  "title": "Aria region",
```

Comment:
I think we should prefix the title here `YouTube:` so it's more specific in the course settings

Reply:
Done.